From 1bde0d0aad6a19eef889fbcc7732be398d8004fd Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:36:57 -0400
Subject: [misc] mei: mei_cl_link remove duplicated check for open_handle_count

Message-id: <1399041470-21330-25-git-send-email-prarit@redhat.com>
Patchwork-id: 79315
O-Subject: [RHEL7.1 PATCH BZ 1088953 24/77] mei: mei_cl_link remove duplicated check for open_handle_count
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit df667a1a2c6bf828c5959a0ba15bcf987a9d17fe
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Mon Oct 21 22:05:41 2013 +0300

    mei: mei_cl_link remove duplicated check for open_handle_count

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Cc: Dean Nelson <dnelson@redhat.com>
Cc: Stefan Assmann <sassmann@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/client.c b/drivers/misc/mei/client.c
index 0ccc22c..87c96e4 100644
--- a/drivers/misc/mei/client.c
+++ b/drivers/misc/mei/client.c
@@ -299,12 +299,6 @@ int mei_cl_link(struct mei_cl *cl, int id)
   return -EMFILE;
  }
 
- if (dev->open_handle_count >= MEI_MAX_OPEN_HANDLE_COUNT) {
-  dev_err(&dev->pdev->dev, "open_handle_count exceded %d",
-   MEI_MAX_OPEN_HANDLE_COUNT);
-  return -ENOENT;
- }
-
  dev->open_handle_count++;
 
  cl->host_client_id = id;
-- 
1.7.1